registry/endpoint.go

```diff
@@ -245,7 +268,7 @@ func (e *Endpoint) pingV2() (RegistryInfo, error) {
 	if resp.StatusCode == http.StatusOK {
 		// It would seem that no authentication/authorization is required.
 		// So we don't need to parse/add any authorization schemes.
-		return RegistryInfo{Standalone: true}, nil
+		return RegistryInfo{}, nil
```

Comment:
I just removed this because a v2 registry no longer has the concept of a "standalone" registry.